src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json

```diff
@@ -1440,8 +1440,19 @@
       }
     }
   },
-  "field-width-medium": {
-    "value": "208px",
+  "field-default-width-small": {
+    "value": "192px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "field-default-width-small",
+        "deprecated-uuids": "096ddcb7-5afd-416f-9a11-4275556c4b52"
```

Comment:
@nabuhasan @GarthDB I noticed a "deprecated-uuids" here, is this expected? I'm not seeing it in any of the other changes.

Reply:
Yeah I saw those as well and was tempted to remove it, but I wasn't sure how they were applied. So I kept them in case this is a part of the automatic workflow. 

I certainly didn't manually add them!

Reply:
That is the UUID for the deprecated `field-width` in the spectrum-tokens repo.

Reply:
@GarthDB How is this used and does this need to be added to any of the other sizes? Just want to make sure we understand! As long as it's expected, it's all good.